src/io/flutter/view/FlutterView.java

```diff
@@ -237,6 +238,16 @@ protected void perform(AnActionEvent event) {
   }
 }
 
+class ToggleInspectModeAction extends AbstractToggleableAction {
```

Comment:
If we decide the behavior is more of a one-shot, open action, this should inherit from `FlutterViewAction`, and the action name change to something like `Show Widget Inspector`.